pkg/controller/deliverable/reconciler_test.go

```diff
@@ -268,7 +268,7 @@ var _ = Describe("Reconciler", func() {
 						Message: "some informative message",
 					},
 				}
-				repo.GetDeliveriesForDeliverableReturns([]v1alpha1.ClusterDelivery{delivery}, nil)
+				repo.GetDeliveriesForDeliverableReturns([]*v1alpha1.ClusterDelivery{&delivery}, nil)
```

Comment:
indentation

Reply:
sorry, wdym? is the suggestion to split the args into multiple lines? the change here was solely making `clustedelivery` a pointer

Reply:
it was showing all the way to left before? maybe bad diff?